modules/ipi-install-node-requirements.adoc

```diff
@@ -37,6 +37,11 @@ Do not deploy a cluster with only one worker node, because the cluster will depl
 * *Network interfaces:* Each node must have at least one network interface for the routable `baremetal` network. Each node must have one network interface for a `provisioning` network when using the `provisioning` network for deployment. Using the `provisioning` network is the default configuration.
 
 * *Unified extensible firmware interface (UEFI):* Installer-provisioned installation requires UEFI boot on all {product-title} nodes when using IPv6 addressing on the `provisioning` network. In addition, UEFI Device PXE Settings must be set to use the IPv6 protocol on the `provisioning` network NIC, but omitting the `provisioning` network removes this requirement.
```

Comment:
It might be outside of the context of this PR, but I will add a suggestion here for reference. "Unified extensible firmware interface" in this line could be updated to "Unified Extensible Firmware Interface".